tests/test.rs

```diff
@@ -142,18 +156,8 @@ fn do_clean(directory: &Path) {
         "default.profraw",
     ];
 
-    let mut glob_builder = GlobSetBuilder::new();
     for to_remove_glob in &to_remove_globs {
-        glob_builder.add(Glob::new(to_remove_glob).unwrap());
-    }
-    let to_remove_globset = glob_builder.build().unwrap();
-
-    for entry in WalkDir::new(&directory) {
-        let entry = entry.expect("Failed to open directory.");
-
-        if to_remove_globset.is_match(&entry.file_name()) {
-            fs::remove_file(entry.path()).unwrap();
-        }
+        rm_files(directory, to_remove_glob);
```

Comment:
You can pass to_remove_globs, so that you actually use the globset feature of having multiple globs in one